cbg_executor_demo/src/cbg_executor_demo/parameter_helper.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 Robert Bosch GmbH
```

Comment:
nitpick here, but do we really need a dedicated library for this one function? Why not just declare it `inline` in the header?